### PR TITLE
Fix Crowdfund + additional issues

### PR DESCRIPTION
**Pull request by Kukks**
* Show free when invoice is fixed and 0 amount in apps
* animations on crowdfund not enabled broke JS
* Redirect to receipt after POS
* Show app link in receipt page
* Do not show 404 page in iframe when using crowdfund and a 0 amount invoice
* Perk expansion in crowdfund was broken

### CODE REVIEW

File: BTCPayServer/Controllers/UIInvoiceController.cs [hunk 1]
```diff
@@ -327,6 +327,10 @@ internal async Task<InvoiceEntity> CreateInvoiceCoreRaw(InvoiceEntity entity, St
                 entity.InternalTags.Add(AppService.GetAppInternalTag(app.Id));
             }
 
+            if (entityManipulator != null)
```
Comment (Kukks) [inline BTCPayServer/Controllers/UIInvoiceController.cs:330]: this is a hack until we switch over to #3353

**Comment by NicolasDorier** [comment]
please can you put a description to what you are actually fixing?

**Comment by Kukks** [comment]
> please can you put a description to what you are actually fixing?

Ah sorry, the commit message didn't get added to the pull request

**Comment by Kukks** [comment]
breaking into small prs

**Pull request closed without merging**